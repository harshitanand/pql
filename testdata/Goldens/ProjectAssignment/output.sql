SELECT "EventId" AS "Id", ("DamageProperty") / (1000) AS "DamagePropertyK" FROM "StormEvents";
