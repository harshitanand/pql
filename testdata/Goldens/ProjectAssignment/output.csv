11032,0
11098,0
60913,6200
11503,2
13913,20
